test/framework/src/main/java/org/elasticsearch/test/disruption/LongGCDisruption.java

```diff
@@ -251,7 +265,11 @@ protected boolean suspendThreads(Set<Thread> nodeThreads) {
                          * assuming that it is safe.
                          */
                         boolean definitelySafe = true;
+                        final long startTime = System.nanoTime();
                         thread.suspend();
+                        if (System.nanoTime() - startTime > TimeUnit.SECONDS.toNanos(3L)) {
+                            sawSlowSuspendBug.set(true);
```

Comment:
Could we make this conditional on the JVM version so that we are sure to keep tracking that JVM bug and to drop this fix when it's no longer needed?

Reply:
Technically we could, but right now we're seeing this failure on all versions. This is a little unexpected on `8` so I figured we'd apply the fix for 8 as well to identify whether it's the same issue (slow `suspend`) there or if it's blocked somewhere else.